server/log_rpc_server.go

```diff
@@ -117,7 +117,7 @@ func (t *TrillianLogRPCServer) QueueLeaves(ctx context.Context, req *trillian.Qu
 			// Return the leaf from the request if it is new.
 			queuedLeaf := trillian.QueuedLogLeaf{
 				Leaf:   req.Leaves[i],
-				Status: &status.Status{Code: int32(code.Code_OK)},
+				Status: &status.Status{},
```

Comment:
It needs a concomitant change in `client/client.go` to watch out for `Status==nil`, but that's worthwhile anyhow.  Done.